sky/optimizer.py

```diff
@@ -1026,7 +1212,8 @@ def _fill_in_launchable_resources(
                     cheapest = feasible_resources[0]
                     # Generate region/zone-specified resources.
                     launchable[resources].extend(
-                        _make_launchables_for_valid_region_zones(cheapest))
+                        _make_launchables_for_valid_region_zones(
+                            cheapest, resources.region_list))
```

Comment:
Instead of adding a new API `region_list` in the `Resources`, can we do a similar thing as list of accelerators, i.e., creating multiple `Resources` object with a single region as candidate in the upper level callers?

Reply:
I think the tricky thing is when user specifies a list for both regions and accelerators. Do we do a cartesian product?

Reply:
same question as next one. commented out.